core/governance/checkpoint.go

```diff
@@ -103,6 +104,15 @@ func (e *Engine) Load(ctx context.Context, data []byte) error {
 			}
 			enct.current = prop.Terms.EnactmentTimestamp
 
+			if prop.NewMarket().Changes.LiquiditySLAParameters == nil {
+				prop.NewMarket().Changes.LiquiditySLAParameters = &types.LiquiditySLAParams{
```

Comment:
don't you want to reuse the default set you already have somewhere else?

Reply:
well, it's the same ones, it's just that when loading a snapshot this is done at another time than when loading a checkpoint.

Reply:
what I mean is define once this: 

```
migrationSLAParams = &types.LiquiditySLAParams{
			PriceRange:                  num.MustDecimalFromString("0.05"),
			CommitmentMinTimeFraction:   num.MustDecimalFromString("0.95"),
			PerformanceHysteresisEpochs: 1,
			SlaCompetitionFactor:        num.MustDecimalFromString("0.90"),
		}

and use in both places

prop.NewMarket().Changes.LiquiditySLAParameters = migrationSLAParams
...
mkt.LiquiditySLAParams = migrationSLAParams
```